.github/workflows/main.yml

```diff
@@ -0,0 +1,94 @@
+name: Playwright Tests
+on:
+  pull_request:
+    branches:
+      - develop
+      - master
+      - release-*
+jobs:
+  
+  linux:
+    name: ${{ matrix.os }} (${{ matrix.browser }})
+    strategy:
+      fail-fast: false
+      matrix:
+        browser: [chromium, firefox]
+        os: [ubuntu-18.04]
+        nodeversion: ['16']
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.nodeversion }}
+    - name: Setup custom host for sdkapp.example.com
+      run: echo "127.0.0.1 sdkapp.example.com sdks.mytestrun.com sdkapp.petrov.ca auth.example.com openig.example.com api.example.com user.example.com rest.example.com bank.example.com openam.example.com spa.example.com sdk.mytestrun.com sdks.mytestrun.com forgerock-sdk.local" | sudo tee -a /etc/hosts
+    - name: Install NPM
+      run: npm i -g npm@8
+    - name: Install Dependencies
+      run: npm ci
+      env:
+        DEBUG: pw:install
+        PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
```

Comment:
We skip browser downloads because in the future, it would be better for the `npx playwright install --with-deps` line to only install the browser from `${{ matrix.browser }}` so we can speed up CI times.

I skipped this in this work here and just installed all the browsers because it was a little bit of a premature optimization but also because I was running into some problems that were not worth debugging to slow this work down. It will be addressed in future work around the pipeline.